PluginsAndFeatures/azure-toolkit-for-intellij/rider/src/org/jetbrains/plugins/azure/functions/run/AzureFunctionsConfigurationProducer.kt

```diff
@@ -74,4 +89,10 @@ class AzureFunctionsConfigurationProducer
         return FileUtil.toSystemIndependentName(item.getFile()?.path ?: "") ==
                 configurationConfiguration.parameters.projectFilePath
     }
+
+    private fun isInProject(projectFilePath: String, childPath: String?): Boolean {
+        if (childPath == null) return false
+
+        return File(childPath).startsWith(File(projectFilePath).parentFile)
```

Comment:
Just curious, is that for case when you are in misk files?

Reply:
Yes, making sure in all cases we know whether the file is in the project path (as the original file _may_  no always be included in the MSBuild project model and exist solely on disk.